modules/ROOT/pages/major-changes.adoc

```diff
@@ -9,6 +9,75 @@ This list is in reverse chronological order to keep recent changes at the top.
+The new defaults will depend on both the CPU architecture and platform.
+
+NOTE: The changes will only affect new Fedora CoreOS installations.
+      Upgraded systems will retain their current console settings.
+
+See also the https://lists.fedoraproject.org/archives/list/coreos-status@lists.fedoraproject.org/thread/GHLXX4MXNHUEAXQLK6BZN45IQYHRVQB4/[coreos-status anouncement].
+
+=== Planning
+
+This change will be rolled out progressively:
+
+|===
+|Update Stream |Targeted release date
+
+|`next` | 2022-10-03 (37.20221003.1.0)
+|`testing`| 2022-11-28
+|`stable`| Will follow `testing` as usual
+|===
+
+=== Notes
+
+The current default depends on the CPU architecture:
+
+- On x86_64, the first serial port ttyS0 is the primary console and the graphical console is secondary.
+
+- On other architectures, Fedora CoreOS generally does not configure a particular console, leaving the bootloader and kernel to follow their own defaults.
+  This typically means that a graphical console is used if one is available, and a serial console otherwise.
+
+The new defaults will depend on both the CPU architecture and platform.
+The exact configuration is in https://github.com/coreos/fedora-coreos-config/blob/next-devel/platforms.yaml[`platform.yaml` (next-devel branch)].
+In summary:
+
+- On many architecture/platform pairs, Fedora CoreOS will allow GRUB and the kernel to follow their own defaults.
+  On x86_64, this causes the graphical console to be selected, even if no video card is available.
+  In particular, *x86_64 bare metal installs will no longer use a serial console by default*.
+
+- On platforms that expect specific system consoles to be used, such as AWS, Azure, and GCP, Fedora CoreOS will select those consoles by default.
+
+- The QEMU image will continue to select ttyS0 as the primary console and the graphical console as secondary.
+
+If the new defaults aren't appropriate for your environment, you can override them in several ways:
+
+- For bare metal machines installed with coreos-installer 0.16.0+, use the `--console` install option to specify the desired consoles using https://www.kernel.org/doc/html/v4.14/admin-guide/serial-console.html[kernel command-line syntax].
+  This will configure both the GRUB bootloader and the kernel.
+  For example, to retain the previous console behavior, install with `--console tty0 --console ttyS0,115200n8`.
+  To customize the --console parameter in a live image, use `iso customize --dest-console` or `pxe customize --dest-console`.
+
+- For machines launched directly from an OS image, use the Butane kernel_arguments section to configure the desired console kernel arguments.
+  Note that this will not affect the consoles used by the bootloader.
+  For example:
+
+[source, yaml]
+----
+variant: fcos
+version: 1.4.0
+kernel_arguments:
+  # Order is significant, so treat these as a single argument
+  should_exist: ["console=tty0 console=ttyS0,115200n8"]
+----
```

Comment:
In the [emergency shell docs](https://docs.fedoraproject.org/en-US/fedora-coreos/emergency-shell/#_configuring_the_console_with_ignition) I ended up adding all the default arguments to `should_not_exist` to be more robust.